yarn.lock

```diff
@@ -1875,31 +1865,32 @@
   dependencies:
     "@oasisprotocol/client" "^0.1.0-alpha5"
 
-"@oasisprotocol/client-signer-ledger@0.1.0-alpha2":
-  version "0.1.0-alpha2"
-  resolved "https://registry.yarnpkg.com/@oasisprotocol/client-signer-ledger/-/client-signer-ledger-0.1.0-alpha2.tgz#361aa2a4bd7345767c758894d501466fe54f9090"
-  integrity sha512-rR9SNUTpBEqLsmbGZYuFl1WgNo0u0G7UAP7WWPwwPnLxNJrIn4g9UjDkrJZ4Oh5Orhp/x9yj+/Al0LlupAtrcg==
+"@oasisprotocol/client-signer-ledger@0.1.0-alpha3":
+  version "0.1.0-alpha3"
+  resolved "https://registry.yarnpkg.com/@oasisprotocol/client-signer-ledger/-/client-signer-ledger-0.1.0-alpha3.tgz#0870cf1d5426d2438708374d0e64fc6a3998c783"
+  integrity sha512-ipaEBRnxeF1MOa7fkHTvJX0ZL5l5Vax76UhEXmGpK78quQGc4DVycTurE5ZcxPlPoIZVDxQimHXlTZMOkuqq1w==
   dependencies:
-    "@ledgerhq/hw-transport-webusb" "^5.41.0"
-    "@oasisprotocol/client" "^0.1.0-alpha3"
-    "@oasisprotocol/ledger" "^0.1.0"
+    "@ledgerhq/hw-transport-webusb" "^6.11.0"
+    "@oasisprotocol/client" "^0.1.0-alpha7"
+    "@oasisprotocol/ledger" "^0.2.0"
 
-"@oasisprotocol/client@0.1.0-alpha5", "@oasisprotocol/client@^0.1.0-alpha5":
-  version "0.1.0-alpha5"
-  resolved "https://registry.yarnpkg.com/@oasisprotocol/client/-/client-0.1.0-alpha5.tgz#b3327883c9bbb50c040943a3649c2f47c75da955"
-  integrity sha512-N0kBdTdPcPMo1iuK8SPXC54Xj2V6eF7MWdguuZ3c93kQeXzmEMEIbJg2KW2qJV/tPICvspnU4N/opbKrnzbAtQ==
+"@oasisprotocol/client@0.1.0-alpha7", "@oasisprotocol/client@^0.1.0-alpha7":
+  version "0.1.0-alpha7"
+  resolved "https://registry.yarnpkg.com/@oasisprotocol/client/-/client-0.1.0-alpha7.tgz#3f4bd8edd3539e2262bdd07027d95e2becc6c011"
+  integrity sha512-JzLkyZ6VuiNrOT6x4XqOsqpMqbIhhMBnt8ve535fq1z7oKpx6+WrkyerI5CeoMX1UR8DKxT3MyMq7Dq8+5caRg==
   dependencies:
     bech32 "^2.0.0"
     bip39 "^3.0.4"
-    cborg "^1.3.4"
-    grpc-web "^1.2.1"
+    cborg "^1.5.2"
+    grpc-web "^1.3.0"
     js-sha512 "^0.8.0"
+    protobufjs "~6.11.2"
     tweetnacl "^1.0.3"
 
-"@oasisprotocol/client@^0.1.0-alpha3":
-  version "0.1.0-alpha4"
-  resolved "https://registry.yarnpkg.com/@oasisprotocol/client/-/client-0.1.0-alpha4.tgz#d5ce6ed2cdf6c9c9a2994634727133ca173c2538"
-  integrity sha512-iXHGbJMwIWGuCDLABQlqkauCnGk6gtaMiurS8+wlhX3IA9QbW4ZL4/tX3mhFCYvSZY/eVHE9FGSF17rhZt0zpQ==
+"@oasisprotocol/client@^0.1.0-alpha5":
+  version "0.1.0-alpha5"
```

Comment:
why won't yarn use 0.1.0-alpha7 for this

Reply:
I think reinstalling @oasisprotocol/client-ext-utils@0.1.0-alpha3 would fix it

Reply:
woa it does, spooky!

thanks